templates/dashboard.jinja.html

```diff
@@ -364,7 +364,9 @@ <h1>
 
   <div class=run-info>
     <ul>
-      <li>{{ run["status"] }}</li>
+      <li>{{ run["status"] }}{% if "__duration_str" in run %}
+      ({{ run["__duration_str"] }})
+      {% endif %}</li>
       <li>{{ run["start_time"] }}</li>
```

Comment:
Can we add `START TIME:`  before `run["start_time"]` ?

Someone, who is viewing a Litani report for the first time, might not understand what the timestamp on the top of the report refers to.